### PR TITLE
Fix keycloak-sync call in GitHub actions

### PR DESCRIPTION
**Pull request by asanchezr**
As part of the move to ArgoCD, all containers have a suffix (ie "-dev", "-prod" etc).
This change fixes the keycloak-sync call for DEV and PROD (the other environments are okay)

### CODE REVIEW

File: .github/workflows/deploy-prod-start-argo.yml [hunk 1]
```diff
@@ -135,7 +135,7 @@ jobs:
...
           oc process -f ./openshift/4.0/templates/jobs/mayan-sync.yaml -p NAMESPACE=3cd915-prod -p TOKEN_URL=https://loginproxy.gov.bc.ca:443/auth/realms/standard/protocol/openid-connect/token -p CLIENT_ID=property-services-project-api-4380 -p MAYAN_SYNC_URL=https://pims-app-3cd915-prod.apps.silver.devops.gov.bc.ca/api/documents/sync/mayan -p KEYCLOAK_SECRET_NAME=pims-api-sso | oc create -f - | grep -oP "(?<=\/)[^\s]*" | (read TASK_NAME; oc wait --for=condition=succeeded taskruns/$TASK_NAME --timeout=80s)
 
-## Call the tekton pipeline that executes the keycloak sync. Dependent on the pims-api being accessible. Can run in parallel with the mayan sync.
+  ## Call the tekton pipeline that executes the keycloak sync. Dependent on the pims-api being accessible. Can run in parallel with the mayan sync.
```
Comment (FuriousLlama) [inline .github/workflows/deploy-prod-start-argo.yml:138]: are these changes necessary? some just seem to be formatting

File: .github/workflows/deploy-prod-start-argo.yml [hunk 2]
```diff
@@ -153,5 +153,4 @@ jobs:
       - name: call scripts to sync keycloak
         shell: bash
         run: |
-          oc process -f ./openshift/4.0/templates/jobs/keycloak-sync-pipeline-run.yaml -p ASPNETCORE_ENVIRONMENT=$ASPNETCORE_ENVIRONMENT -p NAMESPACE=3cd915-prod -p BRANCH=$DESTINATION -p API_URL=http://pims-api:8080/api  | oc create -f - | grep -oP "(?<=\/)[^\s]*" | (read PIPELINE_NAME; oc wait --for=condition=succeeded pipelineruns/$PIPELINE_NAME --timeout=500s)
-
+          oc process -f ./openshift/4.0/templates/jobs/keycloak-sync-pipeline-run.yaml -p ASPNETCORE_ENVIRONMENT=$ASPNETCORE_ENVIRONMENT -p NAMESPACE=3cd915-prod -p BRANCH=$DESTINATION -p API_URL=http://pims-api-prod:8080/api  | oc create -f - | grep -oP "(?<=\/)[^\s]*" | (read PIPELINE_NAME; oc wait --for=condition=succeeded pipelineruns/$PIPELINE_NAME --timeout=500s)
```
Comment (FuriousLlama) [inline .github/workflows/deploy-prod-start-argo.yml:156]: was there anychange to this line?

**Review by FuriousLlama — APPROVED** [review]

File: .github/workflows/deploy-prod-start-argo.yml (hunk 2 above)
Comment (asanchezr) [reply .github/workflows/deploy-prod-start-argo.yml:156]: This is the relevant change
image

File: .github/workflows/deploy-prod-start-argo.yml (hunk 1 above)
Comment (asanchezr) [reply .github/workflows/deploy-prod-start-argo.yml:138]: yes, the auto-format on save on VS code modified the comment indentation but there are actual changes

image
